.jscsrc

```diff
@@ -0,0 +1,7 @@
+{
+    "preset": "wordpress",
+    "fileExtensions": [ ".js" ],
+    "excludeFiles": [
+    	"js/skip-link-focus-fix.js"
```

Comment:
I excluded it as it looked like it came from an external source which does not use the same coding standards as WP.

Fixing the file up is easy enough, but would have to be done again if a newer version of `skip-link-focus.js` at some point would be used.